lib/components/tabs/tabview.coffee

```diff
@@ -110,7 +110,10 @@ module.exports = class KDTabView extends KDScrollView
       minWidth : minHandleWidth
 
     newTabHandle.on 'HandleIndexHasChanged',  @bound 'resortTabHandles'
-    newTabHandle.closeHandler?.on 'click', => @handleCloseAction paneInstance
+
+    unless paneInstance.isDetached
+      newTabHandle.closeHandler?.on 'click', ->
+        paneInstance.parent.handleCloseAction paneInstance
```

Comment:
what is the difference between, `@handleCloseAction` vs `paneInstance.parent` other than it is anti-pattern?